decidim/decidim
# Move rubocop-faker to the decidim-dev module

#### :tophat: What? Why?
It would be nice if we could use the same Rubocop definitions easily in instances as we have in the core. Similar technique what is used today in the bulletin board server application.

This would mean that in Decidim applications, we could do this in the local `.rubocop.yml`:
```yml
inherit_from: https://raw.githubusercontent.com/decidim/decidim/develop/.rubocop.yml
```

And we could avoid maintaining the definitions in our applications.

#### Testing
- Add the described `.rubocop.yml` in a decidim instance where you have `decidim-dev` as a development dependency
- Run `bundle exec rubocop`
- See error

#### :clipboard: Checklist
